app.ts

```diff
@@ -0,0 +1,285 @@
+let fromParameter = 0;
+let timer: number;
+let recordCount: number;
+
+//// On Window Load
+window.onload = function () {
+    getRecordCount()
+        .then(() => {
+            return getHeadings();
+        })
+        .then(() => {
+            return getTable();
+        });
+    console.log("CALL HECTOR!!!");
+};
+
+function getParameters(fromParameter: number) {
+    let toParameter: number;
+    let noOfRows = getNoOfRows();
+
+    toParameter = fromParameter + noOfRows;
+    return toParameter;
+}
+
+function getNoOfRows(): number {
+    const height = window.innerHeight;
+    let noOfRows = Math.floor(height / 40);
+    return noOfRows;
+}
+
+//// Functions To Create/Clear The HTML
+// Heading Row
+function createHeadingRow(headingData: string) {
+    const heading: HTMLElement | null = document.getElementById("heading");
+
+    let headings = `<div class="headings" id="headings">${headingData}</div>`;
+    if (heading !== null) {
+        heading.innerHTML += headings;
+    }
+}
+
+// Table Content
+function createTableContent(contentData: string) {
+    const content: HTMLElement | null = document.getElementById("content");
+
+    let table = `<div id="row-${contentData[0]}" class="rows"></div>`;
+    if (content !== null) {
+        content.innerHTML += table;
+    }
+
+    let rows: HTMLElement | null = document.getElementById("row-" + contentData[0]);
+    for (let x = 0; x < contentData.length; x++) {
+        let rowCols = `<div class="row_cols">${contentData[x]}</div>`;
+        if (rows !== null) {
+            rows.innerHTML += rowCols;
+        }
+    }
+}
+
+//// Fetch Requests
+// Response Error Handling
+function handleErrors(response: Response) {
+    if (!response.ok) {
+        throw Error(response.statusText);
+    }
+    return response;
+}
+
+// Heading Row (Getting the columns data)
+function getHeadings() {
+    return fetch("http://localhost:2050/columns", {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+    })
+        .then(handleErrors)
+        .then((response) => response.text())
+        .then((data) => {
```

Comment:
Please remove the `()` or specify the type as currently the compiler thinks the type is `any`.